### PR TITLE
refactoring, bugfixes and optimizations

### PR DESCRIPTION
**Pull request by okx-code**
- shipping command consistent based on user id
- merge mute and unmute
- refactoring
- replace only first buy in ^shop so ^shop buy buy buy `n` isn't valid

### CODE REVIEW

File: src/main/java/io/github/jroy/happybot/commands/FansCommand.java [hunk 1]
```diff
@@ -15,14 +15,14 @@ public FansCommand() {
 
     @Override
     protected void executeCommand(CommandEvent e) {
-        e.reply("Loading...");
+        e.getChannel().sendTyping().queue();
```
Comment (JRoy) [inline src/main/java/io/github/jroy/happybot/commands/FansCommand.java:18]: OooOOo good idea, forgot we can send typing :3

**Comment by okx-code** [comment]
Why waiting on merge rather than immediate?

**Comment by JRoy** [comment]
@okx-code I am reviewing the pull request, therefore it's pending merge.

File: src/main/java/io/github/jroy/happybot/commands/OgCommand.java [hunk 2]
```diff
@@ -19,13 +19,13 @@ protected void executeCommand(CommandEvent e) {
             Member u = C.getMentionedMember(e);
             if (C.hasRole(u, Roles.OG)) {
                 C.removeRole(u, Roles.OG);
-                e.replySuccess(u.getUser().getAsMention() + " is no OG!");
+                e.replySuccess(u.getUser().getAsMention() + " is no longer OG!");
```
Comment (JRoy) [inline src/main/java/io/github/jroy/happybot/commands/OgCommand.java:22]: This was intentional, but whatever.

File: src/main/java/io/github/jroy/happybot/commands/money/GambleCommand.java [hunk 3]
```diff
@@ -32,15 +32,15 @@ protected void executeCommand(CommandEvent e) {
             e.reply("**Gamble System Overview:**\n" +
                     "You may only gamble away your life savings every 3 minutes to avoid life destruction.\n" +
                     "Patron Boys and Mods+ may gamble their college savings every minute.\n" +
-                    "To start a gamble you can do `^gamble <amount of coins ranging from 100-"+ String.valueOf(RuntimeEditor.getGambleMax()) + ">`\n" +
-                    "You have a 50% chance of you getting your bet  and a 50% chance of your bet being lost.\n");
+                    "To start a gamble you can do " + C.escape("^gamble <amount of coins ranging from 100-"+ RuntimeEditor.getGambleMax() + ">") + "\n" +
```
Comment (JRoy) [inline src/main/java/io/github/jroy/happybot/commands/money/GambleCommand.java:35]: This escape is not needed

**Review by JRoy — APPROVED** [review]
Solid Changes! Merging...

**Pull request merged**